lib/pager_duty/connection.rb

```diff
@@ -39,7 +39,12 @@ def call(env)
           message = "Got HTTP #{response['status']} back for #{url}"
           if error = response.body['error']
             # TODO May Need to check error.errors too
-            message += "\n#{error.to_hash}"
+			begin
+			  message += "\n#{error.to_hash}"
+			rescue NoMethodError => e
+			  message += error
+			end
+
```

Comment:
It's better to check for a method to exist before using it, rather than try it and rescue:

```suggestion
           message += "\n#{error.respond_to?(:to_hash) ? error.to_hash : error}"
```

It also looks like this contribution has tabs, when the rest of the file has spaces.

Reply:
I did some digging with fresh eyes today, and realized that `response.body['error']` is JSON encoded as a string. Has the .to_hash code worked in your env?

Reply:
I also changed line 39 because `response['status']` wasn't working, and I added the reason phrase.